extensions/flow/package.json

```diff
@@ -28,6 +30,13 @@
       "description": "Skip current session",
       "mode": "no-view"
     },
```

Comment:
The `Skip Session` command is somewhat useless but I decided to leave it because I have no way of knowing if current users are executing it

Reply:
Makes sense √